src/tave.fd/CMakeLists.txt

```diff
@@ -10,8 +10,13 @@ endif()
 
 set(exe_name tave.x)
 add_executable(${exe_name} ${fortran_src})
-target_link_libraries(${exe_name} PRIVATE bacio::bacio_4
+if(LINK_W3EMC)
```

Comment:
Not sure why/where this is causing issues.  CMake should be able to resolve the dependencies.  If there is a version change in bacio that depends on w3emc, it should be coded based on version of bacio.

Reply:
Understood. Thanks for explaining.
@DavidBurrows-NCO Please create an issue to remove this section when spack-stack is upgraded to 1.8.0 so that we can revisit it then.

Reply:
@aerorahul @mark-a-potts Issue #88 created